rest/route/middleware.go

```diff
@@ -196,14 +196,10 @@ func (m *canCreateMiddleware) ServeHTTP(rw http.ResponseWriter, r *http.Request,
 	next(rw, r)
 }
 
-// This middleware is more restrictive than checkProjectAdmin, as branch admins do not have access
-func NewRepoAdminMiddleware() gimlet.Middleware {
-	return &projectRepoMiddleware{}
-}
-
 type projectRepoMiddleware struct {
 }
 
+// ServeHTTP is needed despite not being used because because it needs to be implemented for projectRepoMiddleware
```

Comment:
I suspect this can be removed, if you're removing the constructor.

Reply:
// Code generated by legacy codegen tool.
// Manual edits are allowed but please do not replicate this code style.

Reply:
removed